### ISSUE TITLE
ER: Sometimes we use quotes around text in .md files and sometimes we don't

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#7540]
### Dependency
- https://github.com/hackforla/website/issues/7565 (waiting on recommendation.  When we have the recommendation we can proceed with making an epic to audit and then make issues to change the way we use quotes in files to be consistent

### Emergent Requirement - Problem
Inconsistent formatting rules for when to use quotes in markdown files

### Issue you discovered this emergent requirement in
- https://github.com/hackforla/website/issues/6842

https://github.com/hackforla/website/blob/gh-pages/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml?plain

https://github.com/hackforla/website/blob/271a82e89f55bdeeb80677403558228aee2bbe24/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml?plain=1#L224

https://github.com/hackforla/website/blob/271a82e89f55bdeeb80677403558228aee2bbe24/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml?plain=1#L233

### Date discovered 
2024-09-29

### Did you have to do something temporarily
- [ ] YES
- [x] NO

### Who was involved
@ExperimentsInHonesty @kgold2018 

### What happens if this is not addressed
developers are confused

### Recommended Action Items
- [x] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

### Potential solutions [draft]
We need to make an issue to research and recommend best practice, then audit the site for anomalies (making spreadsheet) for new good first issues to fix the quotes issue on each file.

**Comment by HackforLABot** [comment]
Hi @codyyjxn, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by codyyjxn** [comment]
i. Availability: M > F
ii. ETA: EOD

**Comment by ExperimentsInHonesty** [comment]
- Single Issue: https://github.com/hackforla/website/issues/7565
   - Action Item to create decision record
   - [code search for markdown files](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+language%3AMarkdown&type=code)
      - [code search for markdown files with quotes](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%22+language%3AMarkdown&type=code)
- Epic Issue: Audit the site for issues that match recommended changes - we might need to wait on making this issue until the recommendation issue is actually completed, so that we know what steps people need to take in audit.
   - (making spreadsheet)
- Epic Issue: Create issues to X (where X is with or without quotes) for items with X (where X is with or without quotes) quotes

**Comment by ExperimentsInHonesty** [comment]
- revisit this issue after the other issue https://github.com/hackforla/website/issues/7565 gets prioritized

**Comment by HackforLABot** [comment]
Hello @DDVVPP, we appreciate you taking on this issue, however it looks like you're already working on another issue at this time. Please wait until your current issue is merged before taking on another issue. :)

We are going to unassign you from this issue so you can focus on your current issue.

Hfla appreciates you! :)

**Comment by HackforLABot** [comment]
Hi @daras-cu, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by daras-cu** [comment]
### Next Steps
- From #7565:
  - "our Markdown and YAML files should contain single quotes only around strings containing special characters or other data types i.e. numbers or Booleans that we want to treat as strings. This will involve removing some existing unnecessary quotes and possibly changing some double quotes to single."
- We need to create an Epic issue to audit the .md and .yml files in our codebase for the following:
  - Unnecessary quotes
  - Double quotes
  - Missing quotes (in particular, some URLs with colons are missing quotes)
- As part of the Epic, create a Google Sheet with one sheet to list the files to be audited, and a second sheet for auditors to fill in results. Some files may be excluded from the audit if they use Markdown only for formatting, e.g. CONTRIBUTING.md and GHA comment templates. Also create the following templates:
  - Regular issue for auditing a single .md file and adding the findings to the spreadsheet
  - Issue Making issue for creating the audit issues for a block of .md files (probably 10 files per issue)
- See #5248 for a similar Epic
